hashicorp/boundary-ui
# Icu 13302 add filtering pagination support for scopes

✅ Closes: https://hashicorp.atlassian.net/browse/ICU-13302

:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/ICU-13302)

## Description


Add hds::table to Orgs and Projects.
Add search & pagination support.
Refactor usage of `this.store.query('scopes' ...` in all necessary places.

## Screenshots (if appropriate):


https://github.com/hashicorp/boundary-ui/assets/107949262/83a9c025-1053-4f8e-8a09-09b6cf58fdbb



## How to Test


1. start boundary dev env
2. Clear any existing indexedDB from applications tab in dev tools
3. Navigate to Orgs & Projects tab
4. Validate that search works as expected for both list views
5. Validate CRUDL for org & project scopes remains functional



:technologist: [Admin preview](PATH_TO_FEATURE)

:desktop_computer: [Desktop preview](PATH_TO_FEATURE)

## Checklist:
